src/main/java/com/alvarium/annotators/sbom/SbomProvider.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright 2024 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package com.alvarium.annotators.sbom;
+
+public interface SbomProvider {
+  public boolean validate(String filePath) throws SbomException;
+}
```

Comment:
This validates an existing SBoM such as a developer would contribute with a PR. Do we need another method to generate an SBoM from the project dependency file and then compare the contents to be sure the developer provided file is up to date? This is where I think we'll need a `version` property